llvm/lib/Target/AArch64/GISel/AArch64InstructionSelector.cpp

```diff
@@ -901,6 +901,27 @@ static unsigned selectLoadStoreUIOp(unsigned GenericOpc, unsigned RegBankID,
   return GenericOpc;
 }
 
+/// Select the AArch64 opcode for the G_LOAD or G_STORE operation for scalable 
+/// vectors.
+/// \p ElementSize size of the element of the scalable vector
+static unsigned selectLoadStoreSVEOp(const unsigned GenericOpc,
```

Comment:
Is this manually selected in the DAG path? Can you do this in tablegen?

Reply:
https://github.com/llvm/llvm-project/blob/e2ec9514876744627e99fecb7179accdc6969e4e/llvm/lib/Target/AArch64/GISel/AArch64InstructionSelector.cpp#L865-L902

Currently we are already doing `selectLoadStoreUIOp` manually. I'm following the same idea.
Making them select through TableGen would require more work and can be a separate patch.

Reply:
I did some investigation and found that the patch is actually using the existing tablegen-ed patterns in  `selectImpl`. The added instruction selector code is just skipped.